include/libswiftnav/dgnss_management.h

```diff
@@ -22,7 +22,7 @@
 #define DEFAULT_CODE_VAR_KF     (100 * 400)
 #define DEFAULT_AMB_DRIFT_VAR   1e-8
 #define DEFAULT_AMB_INIT_VAR    1e8
-#define DEFAULT_NEW_INT_VAR     1e10
+#define DEFAULT_NEW_INT_VAR     1e4
 
```

Comment:
Perhaps we can put some comments to describe the different initial values?  I don't really understand the difference between "initial" and "new".  Also, these numbers are settings currently in the firmware (whose default value is this preprocesser macro).  It's unrelated to this PR, but should they remain settings as we move towards 1.0?